packages/flutter_tools/lib/src/vscode/vscode.dart

```diff
@@ -326,5 +326,5 @@ class VsCodeInstallLocation {
 
   @override
   // Lowest bit is for isInsiders boolean.
```

Comment:
is this comment still true?

Reply:
I guess that is `edition`, kind of confusing

Reply:
This comment didn't make sense to me, so I just left it.